endorsement/notifications/shared_drive.py

```diff
@@ -103,3 +105,14 @@ def notify_over_quota_non_subsidized_expired():
             drive.save()
         except EmailFailureException as ex:
             pass
+
+
+def notify_admin_missing_drive_count_exceeded(**kwargs):
+    subject = "URGENT: Share Drive Reconcile has too many missing drives"
+    text_template = _email_template("missing_drives.txt")
+    text_message = loader.render_to_string(text_template, kwargs)
+
+    try:
+        send_admin_notification(subject, text_message)
+    except EmailFailureException as ex:
+        pass
```

Comment:
Does a failure log message get emitted elsewhere? If not do you really want this to be silent?

Reply:
yeah, kind of best effort without an internal queue, but you're right, the send failure should at least get logged. thanks!

Reply:
d'oh!  i'm real slow this am... the send_notification call should log both success and failure!